cpp/ql/src/Security/CWE/CWE-020/IRUntrustedDataToExternalAPI.qhelp

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>Using unsanitized untrusted data in an external API can cause a variety of security issues. This query reports 
+external APIs that use untrusted data. The results are not filtered so that you can audit all examples. The query provides data for security reviews of the application and you can also use it to identify external APIs that should be modeled as either taint steps, or sinks for specific problems.</p>
```

Comment:
Please wrap this very long line.

Reply:
Fixed in 8e5e2f864f33a202e47e42bbd3cf41f5aeb3fab8.